File: python_ToSXs/python/04-dayso/26.py
n = 4

result = ""
zsum = 0
multi = 1

for i in range(1, n + 1):
    part = ""
    multi *= i
    zsum += multi

    for j in range(1, i):
        part += str(j) + "x"

    result += part + str(i) + " + "

result = result[0:len(result) - 3]
print(f"{result} = {zsum}")

File: cpp_e9zMZ/cpp/04-dayso/26.cpp
#include <iostream>
using namespace std;

int main() {
    int number = 1;
    int n = 3;
    int multi = 1;
    int sum = 0;
    string result = "";

    while (number <= n) {
        string charater = "";
        multi *= number;
        sum += multi;

        for (int i = 1; i < number; i++) {
            charater += to_string(i) + "x";
        }
        string sign = (number < n) ? " + " : " = ";
        result += charater + to_string(number) + sign;
        number++;
    }
    
    cout << result << sum;
}